File: Model Train/LanguageSpeech.py
result = client.analyze_conversation(

    task={

        "kind": "Conversation",

        "analysisInput": {

            "conversationItem": {

                "participantId": "1",

                "id": "1",

                "modality": "text",

                "language": "en",

                "text": query

            },

            "isLoggingEnabled": False

        },

        "parameters": {

            "projectName": cls_project,

            "deploymentName": deployment_slot,

            "verbose": True

        }

    }

)

File: Model Train/LanguageSpeech.cs
var data = new

{

    analysisInput = new

    {

        conversationItem = new

        {

            text = userText,

            id = "1",

            participantId = "1",

        }

    },

    parameters = new

    {

        projectName,

        deploymentName,

        // Use Utf16CodeUnit for strings in .NET.

        stringIndexType = "Utf16CodeUnit",

    },

    kind = "Conversation",

};

Response response = await client.AnalyzeConversationAsync(RequestContent.Create(data));
